source/agora/node/Node.d

```diff
@@ -45,12 +49,14 @@ public class Node (Network) : API
     /// Reusable exception object
     private RestException exception;
 
+    private GossipProtocol gossip;
```

Comment:
needs a `///` style comment. Every field should have one (for example `exception` above has a comment too).

Reply:
Modified.

Reply:
@AndrejMitrovic,
Check Please~